### PR TITLE
Update accordions.

### LINKED ISSUE #4535: Update the visual design of the section dividers on the Argo item pages (not a design component)
Current design:
image

Proposed design:
image

1. Font size: Semibold 28pt
2. Background color: EAEAEA
3. The corners have a 4pt radius.
4. The carat icon at the end is pointed down when the section is collapsed and pointed up when the section is expanded.

### PR DESCRIPTION
**Pull request by justinlittman**
closes #4535

# Why was this change made?

# How was this change tested?

### CODE REVIEW

**Comment by justinlittman** [comment]
image

File: app/assets/stylesheets/variables.scss [hunk 1]
```diff
@@ -28,11 +29,8 @@ $link-decoration: none; // no underline
 $navbar-dark-color: $white;
 
 // Accordion
-$accordion-button-color: $white;
-$accordion-button-bg: $sonic-silver;
-$accordion-button-active-color: $white;
-$accordion-button-active-bg: $sonic-silver;
-$accordion-button-focus-box-shadow: 0 0 0.25rem 0.25rem rgba($sonic-silver, 0.25);
+$accordion-button-bg: $black10;
```
Comment (jcoyne) [inline app/assets/stylesheets/variables.scss:32]: Consider using CSS properties `--bs-accordion-bg` and `--bs-accordion-active-bg` instead

File: app/assets/stylesheets/variables.scss [hunk 2]
```diff
@@ -11,6 +11,7 @@ $floral-white: #f9f6ef;
 $tahuna-sands: #d2c295;
 $tea: #b6b1a9;
 $silver: #c5c5c5;
+$black10: rgba(black, 0.1);
```
Comment (jcoyne) [inline app/assets/stylesheets/variables.scss:14]: Do we need this?  Can we use `--bs-gray-100` instead?

Comment (justinlittman) [reply app/assets/stylesheets/variables.scss:14]: They're not the same:
image

It seems that `gray-*` is entirely different than `rgba(black, 0.*)`.

File: app/assets/stylesheets/variables.scss (hunk 1 above)
Comment (justinlittman) [reply app/assets/stylesheets/variables.scss:32]: It doesn't look like BS 5.1 has CSS variables for accordions: https://getbootstrap.com/docs/5.1/components/accordion/#variables

File: app/assets/stylesheets/variables.scss (hunk 2 above)
Comment (jcoyne) [reply app/assets/stylesheets/variables.scss:14]: I think it's hard to tell what is comparable if you're using an alpha channel. Why not pick a solid color?

File: app/assets/stylesheets/document-component.scss [hunk 3]
```diff
@@ -1,17 +1,13 @@
 .document-component {
   .accordion-item {
     margin-bottom: 1rem;
+    border-radius: 4px;
```
Comment (jcoyne) [inline app/assets/stylesheets/document-component.scss:4]:
```suggestion
    border-radius: 0.25rem;
```

File: app/assets/stylesheets/document-component.scss [hunk 4]
```diff
@@ -1,17 +1,13 @@
...
+    font-size: 28px;
     font-weight: 500;
     padding: 0.3rem 0.7rem;
-  }
-  .accordion-button.collapsed::after {
-    transform: rotate(-90deg);
-  }
-  .accordion-button:not(.collapsed)::after {
-    transform: rotate(0deg);
+    border-radius: 4px;
```
Comment (jcoyne) [inline app/assets/stylesheets/document-component.scss:10]:
```suggestion
    border-radius: 0.25rem;
```

File: app/assets/stylesheets/variables.scss [hunk 5]
```diff
@@ -10,12 +10,9 @@ $floral-white: #f9f6ef;
 $tahuna-sands: #d2c295;
 
 // Accordion
-$accordion-button-color: $white;
-$accordion-button-bg: $sonic-silver;
-$accordion-button-active-color: $white;
-$accordion-button-active-bg: $sonic-silver;
-$accordion-button-focus-box-shadow: 0 0 0.25rem 0.25rem
-  rgba($sonic-silver, 0.25);
+$accordion-button-bg: var(--stanford-10-black);;
```
Comment (jcoyne) [inline app/assets/stylesheets/variables.scss:13]: Can you set `--bs-accordion-btn-bg` and `--bs-accordion-active-bg` instead on `.document-component`?

**Review by jcoyne — CHANGES_REQUESTED** [review]

File: app/assets/stylesheets/variables.scss (hunk 5 above)
Comment (justinlittman) [reply app/assets/stylesheets/variables.scss:13]:
```
.document-component {
  --bs-accordion-btn-bg: var(--stanford-10-black);
  --bs-accordion-active-bg: var(--stanford-10-black);
}
```
doesn't work. Do I have this rights?

**Review by jcoyne — APPROVED** [review]

**Pull request merged**